### PR TITLE
Support to save file in follow option on ares-log

### PR DESCRIPTION
**Pull request by ssuminahn**
Support to save file in follow option on ares-log

:Detailed Notes:
Modify to support both save and follow options at same time

:Testing Performed:
1. Pass unit test on ose and auto emulator
2. Pass eslint
3. Check below command
  - ares-log -f -s saved.log

:Issues Addressed:
[PLAT-143147] Support to save file in follow option on ares-log

### CODE REVIEW

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/log.js [hunk 1]
```diff
@@ -43,7 +43,11 @@ const util = require('util'),
                     options.session.run(options.cmd, process.stdin, _onData, process.stderr, next);
                 },
             ],  function(err) {
-                next(err, null);
+                if (options.argv.save) {
+                    next(err, {msg : "\nCreated " + options.argv.argv.remain[0] +"\nSuccess"});
```
Comment (YoungeunKim) [inline lib/log.js:47]: minor things : 
+(space)"\nSuccess"

File: lib/log.js [hunk 2]
```diff
@@ -107,7 +110,11 @@ const util = require('util'),
...
```
Comment (YoungeunKim) [inline lib/log.js:114]: minor things : 
+(space)"\nSuccess"

**Review by YoungeunKim — CHANGES_REQUESTED** [review]
Code :-1: 
Verified :+1: 

Just fix minor things.

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: lib/log.js (hunk 2 above)
Comment (ssuminahn) [reply lib/log.js]: I've update this.

File: lib/log.js (hunk 1 above)
Comment (ssuminahn) [reply lib/log.js]: I've update this.

File: lib/log.js [hunk 3]
```diff
@@ -159,7 +166,6 @@ const util = require('util'),
 
                 if (options.argv.save) {
                     fs.writeFileSync(options.argv.argv.remain[0], data, {encoding: 'utf8', flag:'a'});
-                    next(null, {msg : "\nCreated " + options.argv.argv.remain[0] +"\nSuccess"});
```
Comment (YoungeunKim) [inline lib/log.js:162]: Why did you remove this line?

**Review by YoungeunKim — APPROVED** [review]
Code 👍 
Verified 👍

**Pull request merged**